app/src/main/java/org/mozilla/fenix/compose/ThumbnailCard.kt

```diff
@@ -48,43 +51,50 @@ fun ThumbnailCard(
     url: String,
     key: String,
     modifier: Modifier = Modifier,
+    contentDescription: String? = null,
     contentScale: ContentScale = ContentScale.FillWidth,
     alignment: Alignment = Alignment.TopCenter
 ) {
     Card(
         modifier = modifier,
         backgroundColor = colorResource(id = R.color.photonGrey20)
     ) {
-        components.core.icons.Loader(url) {
-            Placeholder {
-                Box(
-                    modifier = Modifier.background(color = FirefoxTheme.colors.layer3)
-                )
-            }
-
-            WithIcon { icon ->
-                Box(
-                    modifier = Modifier.size(36.dp),
-                    contentAlignment = Alignment.Center
-                ) {
-                    Image(
-                        painter = icon.painter,
-                        contentDescription = null,
-                        modifier = Modifier
-                            .size(36.dp)
-                            .clip(RoundedCornerShape(8.dp)),
-                        contentScale = ContentScale.Fit
+        if (inComposePreview) {
+            Box(
+                modifier = Modifier.background(color = FirefoxTheme.colors.layer3)
```

Comment:
I think this `Box` may need the same Modifier treatment treatment as the `Box` in `WithIcon`, so there isn't any size adjustment pop-in. (I can't get this PR to build locally currently because of a Nimbus issue, so can't immediately verify)